homework9/my-app/src/components/SiteNav.js

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+
+function SiteNav() {
+  return (
+    <ul className="list site-nav">
+      <li>
```

Comment:
you have  repeated action for that `li`
create separate component for it like `NavItem`